arcgis-ios-sdk-samples/Content Display Logic/Controllers/ContentTableViewController.swift

```diff
@@ -49,37 +49,24 @@ class ContentTableViewController: UITableViewController {
         self.downloadProgressView = downloadProgressView
     }
 
-    // MARK: - Table view data source
-
-    override func numberOfSections(in tableView: UITableView) -> Int {
```

Comment:
Even though `1` is the default value returned by the class when this method isn't implemented, I have rarely seen this method NOT implemented. An implementation for `numberOfSections(in:)` is almost always provided. I would suggest leaving it.

Reply:
Are you sure? I never include it for single-section tables since it's unnecessary and I don't see how it otherwise adds value.

Reply:
> Are you sure?

Pretty sure. After five years of cocoa development I didn't even know that `UITableViewController` provided an implementation for that method, let alone that the default is `1`. I saw the default implementation used for the first time a month or two ago. Every other time (with the exception of static cells) I've seen the data source provide its own implementation, even just to return `1`. Part of that might be because Xcode's template for a `UITableViewController` subclass provides an implementation that returns `0`. It is easiest to just change `0` to `1`. Anecdotally, the `MasterViewController` class of the Master-Detail app template provides an implementation that returns `1`.

That being said, `UITableViewController` does provide a default implementation that returns `1` when it could throw. So it isn't wrong to use the default implementation here. That's why I made it a suggestion and not an imperative.

Reply:
Interesting. In this case I'd rather just leave it off.